packages/server/src/app.ts

```diff
@@ -154,6 +156,10 @@ export async function initApp(app: Express, config: MedplumServerConfig): Promis
   app.use(attachRequestContext);
   app.use(getRateLimiter(config));
   app.use('/fhir/R4/Binary', binaryRouter);
+
+  // Handle async batch by enqueueing job
+  app.post('/fhir/R4/', authenticateRequest, asyncWrap(asyncBatchHandler(config)));
```

Comment:
I don't think we use trailing slashed on any other routes... It may not matter, but preference for consistency.

Does that mean that this route handler will intercept all other `POST /fhir/R4` requests?

I'm trying to think if there are any others....  Maybe mult-type search by POST?

https://hl7.org/fhir/R4/search.html#Introduction

![image](https://github.com/user-attachments/assets/6aa897d0-3574-4811-8498-c3196fc7a8dd)